app/components/scihist_footer_component.html.erb

```diff
@@ -106,15 +106,6 @@
                                     loading:"lazy"
                             %>
                         </a>
-                        <a href="https://twitter.com/scihistoryorg" class="shi-social__link" target="_blank">
-                            <%= image_tag "from_main_website/twitter-icon.svg",
-                                    width:"22",
-                                    height:"18",
-                                    class:"shi-social__icon",
-                                    alt:"Twitter",
-                                    loading:"lazy"
-                            %>
-                        </a>
```

Comment:
i don't believe the ticket meant to include removing this one?  Double-checking, this link to Institute twitter is still on the main website footer, so I believe we still want to match that here.